### DISCUSSION TITLE
Unselect completion item?

### DISCUSSION THREAD

**Discussion by chtenb** [helix-editor/helix#10976] [Q&A] [answered]
Is it possible to deselect a completion item? Right now it seems I'm stuck choosing at least one completion once I have pressed tab.
Other editors usually allow you to cancel the completion by tabbing back out of the completion list (i.e. completion lists have an implicit null-entry). How do I deselect in Helix? Pressing escape also does not work, it closes the completion list, but the selected item remains.

(Just for reference, I'm testing with https://github.com/estin/simple-completion-language-server)

**Comment by kirawi** [comment, marked as answer]
C-c

**Reply by chtenb** [reply]
According to the keymap that key is bound to comment/uncomment by default. Am I missing something?

**Reply by kirawi** [reply]
That is for normal mode. UI element keybindings (e.g. prompt) take precedence over mode keybindings.

**Reply by chtenb** [reply]
I see, thanks! Does the completion window classify as a prompt? In that case the documentation is incomplete:
It lists `Escape, Ctrl-c | Close prompt`, while escape and ctrl-c have a different behavior

**Reply by the-mikedavis** [reply]
It's a menu documented here https://docs.helix-editor.com/master/keymap.html#completion-menu but currently the C-c binding is missing from the docs
